browser/tor/tor_profile_manager.cc

```diff
@@ -255,4 +255,7 @@ void TorProfileManager::InitTorProfileUserPrefs(Profile* profile) {
   // support extensions in Tor mode and users cannot disable this through
   // settings page for Tor windows.
   pref_service->SetBoolean(translate::prefs::kOfferTranslateEnabled, false);
+
+  // Disable Media Router in Tor to prevent IP leaks when using related APIs.
+  pref_service->SetBoolean(prefs::kEnableMediaRouter, false);
```

Comment:
This doesn't seem to have any effects if it is not set by admin policy.
in `MediaRouterEnabled()`, it checks for `IsManaged()`
```
 const PrefService::Preference* pref = GetMediaRouterPref(context);
  if (pref->IsManaged() && !pref->IsDefaultValue()) {
    CHECK(pref->GetValue()->is_bool());
    bool allowed = pref->GetValue()->GetBool();
    pref_values.insert(std::make_pair(context, allowed));
    return allowed;
  }
  ```

Reply:
Thats why I also added `ItTor` in `MediaRouterEnabled`. This change is not necessary, but increase consistency if this pref is checked directly somewhere.

Reply:
Adding this will just make future maintenance harder especially this does nothing.